planner/core/logical_plan_test.go

```diff
@@ -15,6 +15,7 @@ package core
 
 import (
 	"fmt"
+	"github.com/pingcap/parser/ast"
```

Comment:
Please use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to sort the imports.

Reply:
fixed